fv3core/stencils/d_sw.py

```diff
@@ -828,8 +828,8 @@ def d_sw(
         origin=grid().compute_origin(),
         domain=grid().domain_shape_compute(add=(1, 1, 0)),
     )
-
-    ytp_v.compute(vb, v, ub)
+    ytp_v_obj = ytp_v.YTP_V(spec.namelist)
```

Comment:
I think this naming is somewhat non-intuitive:
- Why do we import `import fv3core.stencils.ytp_u as ytp_u`and then only use its single call?  Would it not be more readable to `from  fv3core.stencils.ytp_u import YTP_U`?
- the fact that this is an object is an implementation detail, right? does this really have to be reflected in the name of `ytp_v_obj`?

Reply:
sure I can do a different import to make it less repetitive. We've been adding 'obj' in the other classes as a convention because we have a lot of 'self.<>' variables. It is still open though for discussion and we can definitely change this convention. I'd rather do that for all the classes we added so far in another PR, if that's ok?

Reply:
sure, just wanted to point that out

Reply:
We are going to remove 'obj' from our naming conventions, just a heads up!